ping-with-dsl/testplans.ts

```diff
@@ -0,0 +1,66 @@
+import { TestPlans } from "dsl/src/dsl"
+import { run } from "dsl/src/testground-runner"
+import goV0230 from "./go/v0.23.0/image.json"
+import rustV0490 from "./rust/image.0.49.0.json"
+
+type PingParams = {
+    max_latency_ms: number,
+    iterations: number,
+}
+
+const testplans: TestPlans<PingParams> = {
+    testPlans: [
+        {
+            name: "Go v0.23",
+            instances: [
+                {
+                    name: "Go v0.23 x Go v0.23",
```

Comment:
I was confused by the format here, I think the instance name and test plan name are meant to be swapped

Reply:
Yeah that's right. Sorry for the confusion. Fixed